extensions/amp-ad-network-adsense-impl/0.1/amp-ad-network-adsense-impl.js

```diff
@@ -138,8 +135,7 @@ export class AmpAdNetworkAdsenseImpl extends AmpA4A {
 
   /** @override */
   delayAdRequestEnabled() {
-    return experimentFeatureEnabled(
-        this.win, ADSENSE_EXPERIMENT_FEATURE.DELAYED_REQUEST);
+    return fastFetchDelayedRequestEnabled(this.win);
```

Comment:
This allows me to keep the experiment name and branches internal within a4a-config (they are currently visibleForTesting)